client/js/components/make-sticky.js

```diff
@@ -1,39 +1,46 @@
 import {nodeList, featureTest} from '../util';
-import debounce from 'lodash.debounce';
-import watch from 'redux-watch';
+import {store$} from '../store';
+import dropRepeats from 'xstream/extra/dropRepeats';
+import {windowOrientationChange$, windowResize$} from '../utils/dom-events';
+import sampleCombine from 'xstream/extra/sampleCombine';
 
 const initialPxFromTop = 15; // TODO: remove magic
-const makeSticky = (els, store) => {
+
+const makeSticky = (els) => {
```

Comment:
I had read [here](https://www.sitepoint.com/redux-without-react-state-management-vanilla-javascript/) about pitfalls of importing the store into components on a case-by-case basis rather than creating it at the top and passing it into components that need it. I'm not sure if we'll be susceptible to the issues outlined in that article (circular dependencies), here?